<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-067IT145" name="Test2" tests="2" failures="1" timestamp="12 Mar 2017 01:44:30 GMT" time="43.839" errors="0">
  <testcase name="Intro" time="6.158" classname="tc.eobao.rbs.RBS_ST1"/>
  <testcase name="BusinessDetails" time="8.715" classname="tc.eobao.rbs.RBS_ST1">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at tc.eobao.rbs.RBS_ST1.BusinessDetails(RBS_ST1.java:82)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- BusinessDetails -->
</testsuite> <!-- Test2 -->
